docs/markdown/reference/starter-templates/README.md

```diff
@@ -111,7 +111,7 @@ created when you run `yarn build`.
 
 ### Importing individual components and directives
 
-As an example, you can import `<b-card>` (plus some of it's sub components) and `<b-table>` as
+As an example, you can import `<b-card>` (plus some of its sub components) and `<b-table>` as
```

Comment:
In the case of "possive" it, the official contraction is "its" (no apostrophe), while "it is" is contacted to "it's". Unlike possessive names like "Bob's" (it's a weird english rule)